static/js/overlays.js

```diff
@@ -77,6 +82,7 @@ export function active_modal() {
 }
 
 export function open_overlay(opts) {
+    scroll_bar.disable_scrolling();
     popovers.hide_all();
```

Comment:
Some overlays use `micromodal` library, we should disable scrolling in them too.